agent/strategy/quality/techdebt/pmdcpd.py

```diff
@@ -0,0 +1,27 @@
+from agent.utils import run_logged
+import os
+
+class PmdCpd:
+    def run(self, directories, techdebt_dir):
+
+        languages = ["cpp","cs","ecmascript","java","objectivec","php","ruby", "scala"]
+        pmdcpdparser_output_folder_path = os.path.join(techdebt_dir,'PMDCPDParser')
+        if not os.path.exists(pmdcpdparser_output_folder_path):
+            os.makedirs(pmdcpdparser_output_folder_path)
+        for language in languages:
+            # Run for each language parser
+            #print(f"Running for language {language}")
+            with open(os.path.join(pmdcpdparser_output_folder_path,language +".txt"), "w") as output_file:
```

Comment:
```suggestion
            with open(pmdcpdparser_output_folder_path / f"{language}.txt", "w") as output_file:
```

Similar to above suggestion.